Automattic/jetpack
# WP 5.5 Compat: Align Jetpack and Core's plugin autoupdates

Fixes #16600
See #15388

#### Changes proposed in this Pull Request:
* Convert the existing `Jetpack_Option` autoupdate plugin list to use new Core option. This means existing autoupdating plugins will be included in the Core update.
* Sync the network option.
* Enable listeners for network options off of multisite. They aren't exclusive to multisite and are, in effect, the same as regular options when used on a single site.

#### Jetpack product discussion
n/a

#### Does this pull request change what data or activity we track or use?
n/a

#### Testing instructions:
* Existing Jetpack site before this patch running on WP 5.4.
* Visit wp-admin/plugins.php?calypsoify=1
* Enable plugin autoupdates for something.
* Via wp.com sandbox, get the `jetpack_autoupdate_plugins` option. Verify it matches.
* Upgrade to WP 5.5 (via the WP beta tester).
* Upgrade to this patch.
* Visit the wp-admin/plugins.php?calypsoify=0 and confirm the Core UI shows the same plugins enabled.
* Visit wp-admin/plugins.php?calypsoify=1 and ensure you're only seeing on option to auto-update (the Core version).
* Make changes.
* Via wp.com sandbox, get the `jetpack_autoupdate_plugins` option. Verify it matches.

#### Proposed changelog entry for your changes:
* TBD
